packages/example-react/project/app/App.js

```diff
@@ -1,25 +1,19 @@
 import React, { Suspense, lazy } from 'react';
 import { Router } from '@reach/router';
 import { configure } from 'mobx';
-import { Provider } from 'mobx-react';
-import { appStore, stateStore } from './stores';
 import AuthorizationRequest from './Request';
 
 const AuthorizationResponse = lazy(() => import(/* webpackChunkName: "Response" */ './Response'));
 
-const stores = { appStore, stateStore };
-window.stateStore = stateStore;
 configure({ enforceActions: 'observed' });
 
 const App = () => (
-  <Provider {...stores}>
-    <Suspense fallback={<div>Loading...</div>}>
-      <Router basepath="/" style={{ height: '100%' }}>
-        <AuthorizationRequest path="/" />
-        <AuthorizationResponse path="/response" />
-      </Router>
-    </Suspense>
-  </Provider>
+  <Suspense fallback={<div>Loading...</div>}>
+    <Router basepath="/" style={{ height: '100%' }}>
+      <AuthorizationRequest path="/" />
+      <AuthorizationResponse path="/response" />
+    </Router>
+  </Suspense>
```

Comment:
Isn't Suspense about lazy-loading routes similar to react-loadable
How does this change relate to the commit message for the pull request?

It is mentioned: Uses the ~ key as the PR for @ alongside scoped packages such as @reach/router hasn't been merged. PR

Are the changes related to @reach/router an example of the tilda working correctly?
Otherwise it seems like that logic should be in a separate pr.

I understand this is 6 commits in total. And not just the Pr message.
Maybe we can update the pr message to include some of these additional changes in the summary that the pull request provides.

Reply:
Not sure I am understanding the first part of your question. `Suspense` is already a feature of this example package and I think it was just moved around to look like it was part of this commit.

Scoped packages should be able to work alongside relative packages that use something like babel-plugin-import.
So I should be able to do something like:
```jsx
import { Router } from '@reach/router';
import { Header, Footer }  '@/components';
```

However if you currently use the `@` symbol its going to think its a relative route and will not resolve to `node_modules` folder. 

Lastly, if you check the angular convention commits you can see the structure that our repos as well as a lot of other open source repos follow: [angular-conventional-commits](https://www.conventionalcommits.org/en/v1.0.0-beta.2/#specification). With that being said the hope was to clean up some of the existing packages while adding this feature in. We don't follow a super strict guideline on what the exact changes that need to happen are scoped only to that PR. As long as all commits follow the conventional commit strategy, the changelog as well as the version bumps via lerna will happen accordingly. 

We have pre-commit hooks that require your commit messages to follow a guideline although it can be passed by using `--no-verify`. I would look into using this as right now I don't think your previous commits to over PR's we have been speaking in follow that convention.